codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -603,9 +635,122 @@ impl Field {
         // TODO: should we dis-allow indexed fields that are either `id` or derived From fields?
         let is_indexed = indexed_count > 0;
 
-        let field_type =
-            FieldType::from_obj_field_type(&field.field_type, derived_from_field, is_indexed)
-                .context(format!("Failed parsing field {}", field.name))?;
+        // Collect directives
+        let decimal_precision_directives = field
+            .directives
+            .iter()
+            .filter(|&directive| directive.name == "precision")
+            .collect::<Vec<&Directive<'_, String>>>();
+        let numeric_directives = field
+            .directives
+            .iter()
+            .filter(|&directive| directive.name == "numeric")
+            .collect::<Vec<&Directive<'_, String>>>();
+
+        // Validate directive usage
+        if decimal_precision_directives.len() > 1 || numeric_directives.len() > 1 {
+            return Err(anyhow!(
+                "EE202: Cannot use more than one of the same directive on field {}",
+                field.name
+            ));
+        }
```

Comment:
Maybe we can generalize this check for other directives?

Reply:
Done :+1:  I had to reorder and move this code higher up, but I think it has a better flow now.